design/Bootstrap_design.Rmd

```diff
@@ -193,13 +193,19 @@ Add section 'Report 4: Bootstrapping' in the maic_tte_unanchor() function of the
   res[["boot_hist"]] <- grDevices::recordPlot()
   res[["HR_median"]] <- HR_median
   res[["boot_ci_HR"]] <- boot_ci_HR
-  res[["boot_ci_HR_BCA"]] <- boot_ci_HR_BCA`
+  res[["boot_ci_HR_BCA"]] <- boot_ci_HR_BCA
 
 ```
 
 
 # Another way
 
+- Changed the old names to the names we are using in maicplus to i.e. intervention_data is no longer used
+- We removed dplyr dependence. So, we can no longer use dplyr::bind_rows and dplyr::mutate
+- I think the function can be reused for anchored setting. In anchored setting, we might not need to merge the pseudo data i.e. if we use HR from the reported comparator study. This should be discussed more thoroughly though.
+- Added couple stop statements for heuristic checks.
+
```

Comment:
@MikeJSeo Added your text here so that it is more clear what has been done and what is the difference from Method 1.